lib/linguist/languages.yml

```diff
@@ -8330,3 +8330,12 @@ xBase:
   tm_scope: source.harbour
   ace_mode: text
   language_id: 421
+Vento:
+  type: markup
+  color: "#ff0080"
+  extensions:
+  - ".vto"
+  - ".vento"
+  tm_scope: text.html.js
```

Comment:
This is not the scope from the grammar. It should be.

Reply:
I run the `script/add-grammar` passing the vscode plugin https://github.com/oscarotero/vscode-vento as argument. I guess it's not 100% compatible and needs some changes.